lib/zenaton/services/properties.rb

```diff
@@ -1,7 +1,13 @@
 # frozen_string_literal: true
 
 require 'singleton'
-require 'json/add/core'
+require 'json/add/date'
```

Comment:
`require 'json/add/core'` is a shortcut for loading these files + `'json/add/symbol'` which this PR removes